src/panels/lovelace/editor/config-elements/glance-card/glance-options.ts

```diff
@@ -0,0 +1,29 @@
+import { ConfigEntity } from "../../../cards/hui-glance-card";
+
+export class GlanceOptions {
+  private entityConf?: ConfigEntity;
+
+  constructor(entityConf: ConfigEntity) {
+    this.entityConf = entityConf;
+  }
+
+  get _name(): string {
```

Comment:
Don't prefix it with `_`, that indicates it's for private use within the class, and that's not here